docs/tutorials/operations.rst

```diff
@@ -549,7 +542,7 @@ DFFML, we need to register them with Python's ``entry_points`` system.
 **setup.py**
 
 .. literalinclude:: /../examples/shouldi/setup.py
-    :lines: 17-28
+    :lines: 17-26
 
 Re-install the package via pip to make registrations take effect.
 
```

Comment:
Re-run though the `Visualizing the DataFlow` section and update output there.

For regenerating the `shouldi-dataflow-processing.svg`, you'll add `-stage processing` to the command from visualizing.

`dffml dataflow diagram -stage processing -simple shouldi/deploy/df/shouldi.json`

Reply:
Let's also update the bullet point list and move

![image](https://user-images.githubusercontent.com/5950433/82928142-abf17580-9f36-11ea-929d-bc78806dfb4f.png)

Reply:
done, thanks